e2e-tests/src/main/java/uk/nhs/adaptors/gp2gp/MessageQueue.java

```diff
@@ -13,13 +13,23 @@
 import javax.naming.InitialContext;
 import javax.naming.NamingException;
 
+import io.netty.util.internal.StringUtil;
```

Comment:
oh just spotted this. package name is "internal" which means we're not meant to use it.